src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -10,6 +10,8 @@
 import java.util.function.BooleanSupplier;
 import java.util.function.Consumer;
 
+import com.google.common.collect.ImmutableList;
+
```

Comment:
This unused import can be removed, since `ImmutableList#copyOf` was replaced with `List#copyOf`.